docker/README.md

```diff
@@ -30,15 +30,67 @@ It's based on the official image, but in order to supply our own `cassandra.yml`
 
 Note that `docker-compose up` will build this for you if you skip this step and  the image hasn't been built before. If you make changes and want to rebuild, you can also skip this step and just include  the `--build` argument to `docker-compose up`, which will force rebuilding the Cassandra image.
 
-### start all services
-
-    docker-compose -f ./docker/docker-compose.yml up
-
-### stop all services
-
-    docker-compose -f ./docker/docker-compose.yml down
-
-This is basically the same as Ctrl+C after running `docker-compose [...] up`, _but_ this does not cleanup/delete any data or temporary files. In fact, the Docker volumes, network and other resources ~~will~~ may hang around so if you are having trouble starting up services because of suspected lingering  data, you need to invoke the above command with the  `--volumes` option (just add it  to the end of the `down` command). This will, apparently, cleanup volumes and give you fresh, empty ones.
+### Services start
+
+All services and apps (exclude kafka, kafka-manager, zookeeper) available in 2 dc (dc1 and dc2).
+
+Related services:
+
+- cassandra
+- zookeeper
+- kafka
+- kafka-manager
+
+Available apps:
+
+- emodb-web
+- emodb-blob
+- emodb-megabus
+- emodb-stash
+
+To start selected services use command:
+
+```
+docker-compose up -d service_name
+```
+
+example: will launch 2 C* nodes, kafka, zookeeper, 2 web and 2 megabus. Check with `docker ps`
```

Comment:
this is not accurate, only 1 C* instance is launched.
here and below remove C* dc2 references.